userguide/tutorials/metrics-newrelic.adoc

```diff
@@ -0,0 +1,172 @@
+development. The multitier option runs Kill Bill and Kaui on two or more
+separate EC2 instances, and uses an AWS RDS database manager along with
+an AWS load balancer. This is an attractive option for many production
+uses.
+
+You should be running Kill Bill version 2021.9 or later. This version
+can be configured to provide its full collection of Kill Bill metrics to
+New Relic (and several other analysis tools). If you open the Kill Bill
+admin page in your browser, it should show that you are
+running the "Kill Bill Premium edition":
+
+image::metrics-kb-admin.png[align=center]
+
+
+If you see this page you are ready to go. If not, you should upgrade
+your installation.
+
+While you are on this page, if you want to see all the metrics that Kill
+Bill has to offer, just add the path `/1.0/metrics` to the URL. Have fun!
+If you find these hard to understand, though, maybe New Relic can help.
+
+== Set Up New Relic
+
+You may already be familiar with New Relic. If not, this section will
+explain how to get started.
+
+First, go to the https://www.newrelic.com/[New Relic website]. New Relic
+offers a free account as long as your usage does not exceed certain limits. Provide a name and email address to sign up.
+
+You should see the New Relic main page. To get back to this page at a future time you can click the words *New Relic One* in the upper left corner.
+
+To set up New Relic to manage the metrics available from Kill Bill, there
+are just two steps: (1) install the New Relic agent in Kill Bill, and (2)
+edit a configuration file to enable the transfer of the complete Kill Bill metrics.
+
+== Install the Agent
+
+The first step in integrating an analysis tool (like New Relic) and a
+source of data to analyze (like Kill Bill) is to set up a means to get
+the source to send its data to the analyzer. New Relic does this by
+installing the *New Relic Agent* in the system producing the data.
+
+1. In the main New Relic page, select the button labeled *Add Some Data*. If this button does not appear, select *+ Add More Data* in the upper right corner. 
+2. An overlay pane should appear. At the top of this pane select *Guided Install*.
+3. The next pane is headed *Installation Plan*. Make sure the *Auto-Discovery* tab is selected, then choose *Linux*.
+4. The next pane summarizes what you have selected. Click on *Begin Installation*. You will see the following page:
+
+image::metrics-newrelic-installation.png[align=center]
+
+5. In the box containing the install command, click *Copy Command*. The command will be copied to your clipboard.
+
+[NOTE]
+This command includes the API key assigned to you by
+New Relic. This authorizes the agent to do its job. This is your *user* API Key, and should not be confused with the key you will use in the next step.
+
+To install the agent: 
+
+1. Login to your EC2 instance. 
+2. Paste the saved install command to the Ubuntu command line. 
+3. Press Enter. The installation should begin.
```

Comment:
```suggestion
.. Login to your EC2 instance. 
.. Paste the saved install command to the Ubuntu command line. 
.. Press Enter. The installation should begin.
```

I think this ^^ should render as sub-steps a, b, and c.

Reply:
Fine. I did not know how to force step numbers and substeps.